MonoGame.Framework.Content.Pipeline/H264Importer.cs

```diff
@@ -16,5 +19,10 @@ public override VideoContent Import(string filename, ContentImporterContext cont
             var content = new VideoContent(filename);
             return content;
         }
+
+        public override VideoContent Import(Stream input, string virtualFilename, ContentImporterContext context)
+        {
+            throw new NotSupportedException("Video content must be stored on disk, not imported from a stream.");
```

Comment:
This isn't ideal; I'm assuming MonoGame expects developers to copy out the video files (and audio files) as-is when compiling content, since the importer doesn't actually import the file into any form of memory structure (VideoContent and AudioContent just store the file path).

Alternatively, we could add a `Stream` property to VideoContent and AudioContent so that the processor can read from that if present (or we read the stream and embed it into the XNB; in my case the processed data never makes it into an XNB).